src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```diff
@@ -0,0 +1,161 @@
+        bottom: 0;
+        left: 0;
+        content: "";
+        border: var(--pf-c-clipboard-copy__group-toggle--BorderWidth) solid;
+        border-color: var(--pf-c-clipboard-copy__group-toggle--BorderTopColor) var(--pf-c-clipboard-copy__group-toggle--BorderRightColor) var(--pf-c-clipboard-copy__group-toggle--BorderBottomColor) var(--pf-c-clipboard-copy__group-toggle--BorderLeftColor);
+      }
+
+      &:hover,
+      .pf-m-hover {
+        &::before {
+          --pf-c-clipboard-copy__group-toggle--BorderBottomColor: var(--pf-c-clipboard-copy__group-toggle--hover--BorderBottomColor);
+        }
+      }
+
+      &:active,
+      &.pf-m-active {
+        &::before {
+          --pf-c-clipboard-copy__group-toggle--BorderBottomColor: var(--pf-c-clipboard-copy__group-toggle--active--BorderBottomColor);
+
+          border-bottom-width: var(--pf-c-clipboard-copy__group-toggle--active--BorderBottomWidth);
+        }
+      }
+    }
+
+    .pf-c-clipboard-copy__group-copy {
+      position: relative;
+      padding-right: var(--pf-c-clipboard-copy__group-copy--PaddingRight);
+      padding-left: var(--pf-c-clipboard-copy__group-copy--PaddingLeft);
+      border: none;
+
+      &::before {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        content: "";
+        border: var(--pf-c-clipboard-copy__group-copy--BorderWidth) solid;
+        border-color: var(--pf-c-clipboard-copy__group-copy--BorderTopColor) var(--pf-c-clipboard-copy__group-copy--BorderRightColor) var(--pf-c-clipboard-copy__group-copy--BorderBottomColor) var(--pf-c-clipboard-copy__group-copy--BorderLeftColor);
+      }
+
+      &:hover,
+      .pf-m-hover {
+        &::before {
+          --pf-c-clipboard-copy__group-copy--BorderBottomColor: var(--pf-c-clipboard-copy__group-copy--hover--BorderBottomColor);
+        }
+      }
+    }
+
+    .pf-c-clipboard-copy__group-text {
+      position: relative;
+      max-width: var(--pf-c-clipboard-copy__group-text--MaxWidth);
+      padding: var(--pf-c-clipboard-copy__group-text--PaddingTop) var(--pf-c-clipboard-copy__group-text--PaddingRight) var(--pf-c-clipboard-copy__group-text--PaddingTop) var(--pf-c-clipboard-copy__group-text--PaddingLeft);
+      overflow: hidden;
+      font-size: var(--pf-c-clipboard-copy__group-text--FontSize);
+      line-height: var(--pf-c-clipboard-copy__group-text--LineHeight);
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      background-color: var(--pf-c-clipboard-copy__group-text--BackgroundColor);
+      border: var(--pf-c-clipboard-copy__group-text--BorderWidth) solid;
+      border-color: var(--pf-c-clipboard-copy__group-text--BorderTopColor) var(--pf-c-clipboard-copy__group-text--BorderRightColor) var(--pf-c-clipboard-copy__group-text--BorderBottomColor) var(--pf-c-clipboard-copy__group-text--BorderLeftColor);
+    }
+  }
+  /* stylelint-enable */
+
+  .pf-c-clipboard-copy__expanded-content {
+    position: absolute;
```

Comment:
Just want to make sure we've confirmed that this should behave like a dropdown with an absolutely positioned expanded content box? And when this box expands, it shouldn't shift the content below it, and should display on top of that content instead? Assuming so, should we also have a `.pf-m-top` modifier in case this box is at the bottom of the page and you want the menu to expand upward?

Reply:
FWIW, the pf3 angular implementation doesn't use absolute positioning if you want to see what that looks like. https://www.patternfly.org/patternfly-ng/#/blockcopy